prow/config.yaml

```diff
@@ -212,8 +212,18 @@ tide:
     - do-not-merge/work-in-progress
     - needs-ok-to-test
     - needs-rebase
+  - repos:
+    - istio/operator
+    labels:
+    - cla: "yes"
```

Comment:
`"cla: yes"` Otherwise yaml thinks you're doing `[{"cla": "yes"}]` rather than `["cla: yes"]`